examples/kotlin-diktat/sarif-actual/save.toml

```diff
@@ -0,0 +1,19 @@
+[general]
+    tags = ["warn", "sarif"]
+    description = "Test warnings discovered by diKTat, reading expected warnings from SARIF file"
+    suiteName = "Only Warnings: with SARIF format"
+    expectedWarningsPattern = "// ;warn:?(.*):(\\d+):? ?(.+)?"
+
+
+[warn]
+    execFlags = "--reporter=sarif"
```

Comment:
hm, I am reading from stdout here...
This is actual result, parsed by the SAVE

Reply:
Yes, but shouldn't we support reading from file here too? I think, for many tools it makes more sense to write JSON report into a file rather than stdout

Reply:
yes, sure, will support

Reply:
@petertrr if you don't mind - I will move it to a separate PR and issue

Reply:
OK, sure